documentation/asciidoc/topics/proc_creating_clusters.adoc

```diff
@@ -20,6 +20,13 @@ For example, create a {brandname} cluster with two pods as follows:
 {kube_client} infinispan create cluster --replicas=3 -Pservice.type=DataGrid infinispan
 ----
 +
+[TIP]
+====
+Add the `--version` argument to control the {brandname} version of your cluster.
```

Comment:
You have spotted a missing feature here :slightly_smiling_face:

I need to add the `--version` arg to the CLI, I've created [ISPN-13936](https://issues.redhat.com/browse/ISPN-13936) to track this

We should leave this in the docs though, as this will be available in the next Operator/Server release.